conf/schemas.go

```diff
@@ -116,6 +117,19 @@ func ReadSchemas(file string) (Schemas, error) {
 		}
 		schema.Priority = int64(p)<<32 - int64(i) // to sort records with same priority by position in file
 
+		reorderBufferStr := sec.ValueOf("reorderBuffer")
+		if len(reorderBufferStr) > 0 {
+			reorderWindow, err := strconv.ParseUint(reorderBufferStr, 10, 32)
```

Comment:
in the code we talkabout "reorderWindow" everywhere for this parameter, but the setting in the config file is reorderBuffer, why? I think it should be reorderWindow also. especially since later more config options could follow for the reorderBuffer. and "reorderBuffer" is the thing doing the reordering, not the setting for the window size